plugins/importexport/datacite/filter/DataciteXmlFilter.inc.php

```diff
@@ -192,7 +192,7 @@ function &process(&$pubObject) {
 		if ($sizesNode) $rootNode->appendChild($sizesNode);
 		// Formats
 		if (!empty($galleyFile)) {
-			$format = Services::get('file')->fs->getMimetype(Services::get('file')->getPath($galleyFile->getData('fileId')));
+			$format = Services::get('file')->fs->getMimetype($galleyFile->getData('path'));
```

Comment:
Oh I must have missed this before, but we're trying not to use `->fs->getMimetype` now. You should have the mimetype in the submission file like this:

```
$format = $galleyFile->getData('mimetype');
```